db/migrate/20150512015621_add_userstamps_to_course_users.rb

```diff
@@ -0,0 +1,14 @@
+class AddUserstampsToCourseUsers < ActiveRecord::Migration
```

Comment:
Oh yeah, seems that the creator will always be the one who submit the enrol request ?